frame/support/procedural/src/construct_runtime/expand/origin.rs

```diff
@@ -68,14 +68,39 @@ pub fn expand_outer_origin(
 	}
 
 	let system_path = &system_pallet.path;
+
 	let system_index = system_pallet.index;
 
+	let system_path_name = system_path.module_name();
+
+	let doc_string = get_intra_doc_string(
+		" Origin is always created with the base filter configured in",
```

Comment:
Can we please get rid of the leading whitespaces in all these codes.

Reply:
Alright. Will do

Reply:
@bkchr i have done that

Reply:
Ty